README.md

```diff
@@ -67,16 +67,16 @@ Read the [installation guide](https://kueue.sigs.k8s.io/docs/installation/) to l
 
 ## Usage
 
-A minimal configuration can be set by running the [examples](examples):
+A minimal configuration can be set by running the [examples](site/static/examples):
 
 ```shell
-kubectl apply -f examples/admin/single-clusterqueue-setup.yaml
+kubectl apply -f site/static/examples/admin/single-clusterqueue-setup.yaml
```

Comment:
Actually, this (and the one below) don't require adjusments.

Reply:
Ah yes the symlink would work fine in a terminal wouldn't it

Reply:
Yeah, from my testing this works fine, let's focus on what is broken only